frameworks/PHP/symfony/symfony.dockerfile

```diff
@@ -5,30 +5,20 @@ ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update -yqq && apt-get install -yqq software-properties-common > /dev/null
 RUN LC_ALL=C.UTF-8 add-apt-repository ppa:ondrej/php
 RUN apt-get update -yqq > /dev/null && \
-    apt-get install -yqq nginx git unzip php7.3 php7.3-common php7.3-cli php7.3-fpm php7.3-mysql  > /dev/null
-RUN apt-get install php7.3-xml  > /dev/null
+    apt-get install -yqq nginx git unzip php7.3 php7.3-common php7.3-cli php7.3-fpm php7.3-mysql > /dev/null
```

Comment:
If you delete only 1 space, like in this line.
The docker cache will not work. :(

Reply:
Only in the php directory, exist 47 dockerfiles.
Using cache each container is build in 2-6 seconds.
Without cache 2-4 minutes.

If only 10 dockerfiles break the docker cache, it's about 40 minutes to build the container vs 40 seconds.

Reply:
i was planning to create separate pr for the rest of the docker files, but that pr can include this one...

Reply:
here are the other variations of `ext-mbstring` installation so there are also candidates for fixing for the docker cache

```
        cakephp.dockerfile  (1 usage found)
            8     apt-get install -yqq nginx git unzip php7.3 php7.3-common php7.3-cli php7.3-fpm php7.3-mysql php7.3-xml php7.3-mbstring php7.0-mcrypt php7.3-intl > /dev/null
        fuel.dockerfile  (1 usage found)
            8     apt-get install -yqq nginx git unzip php7.3 php7.3-common php7.3-cli php7.3-fpm php7.3-mysql php7.3-xml php7.3-mbstring php7.0-mcrypt  > /dev/null
        hhvm.dockerfile  (1 usage found)
            10     apt-get install -yqq hhvm nginx git unzip php7.3 php7.3-common php7.3-cli php7.3-fpm php7.3-mysql php7.3-xml php7.3-mbstring php7.3-mongodb  > /dev/null
        laravel.dockerfile  (1 usage found)
            9 RUN apt-get install -yqq php7.3-mbstring php7.3-xml  > /dev/null
        lumen.dockerfile  (1 usage found)
            9 RUN apt-get install -yqq php7.3-mbstring php7.3-xml  > /dev/null
        php-eloquent.dockerfile  (1 usage found)
            8     apt-get install -yqq nginx git unzip php7.3 php7.3-common php7.3-cli php7.3-fpm php7.3-mysql  php7.3-mbstring > /dev/null
        php-laravel-query-builder.dockerfile  (1 usage found)
            8     apt-get install -yqq nginx git unzip php7.3 php7.3-common php7.3-cli php7.3-fpm php7.3-mysql  php7.3-mbstring > /dev/null
        symfony-raw.dockerfile  (1 usage found)
            9 RUN apt-get install -yqq php7.3-mbstring php7.3-xml  > /dev/null
        symfony.dockerfile  (1 usage found)
            9 RUN apt-get install -yqq php7.3-mbstring php7.3-xml  > /dev/null
        yii2-raw.dockerfile  (1 usage found)
            8     apt-get install -yqq nginx git unzip php7.3 php7.3-common php7.3-cli php7.3-fpm php7.3-mysql php7.3-mbstring  > /dev/null
        yii2.dockerfile  (1 usage found)
            8     apt-get install -yqq nginx git unzip php7.3 php7.3-common php7.3-cli php7.3-fpm php7.3-mysql php7.3-mbstring  > /dev/null
        zend.dockerfile  (1 usage found)
            9 RUN apt-get install -yqq php7.3-xml php7.3-mbstring  > /dev/null
```

Reply:
I have a PR ready to send to optimize docker cache.
But I wait for your changes or I will break your PR.

Reply:
can we create a list of desired php extensions and fix the rest of the dockerfiles (in another pr)?